.gitignore

```diff
@@ -1 +1,2 @@
 .DS_Store
+crytic-export
```

Comment:
This change is unrelated to this PR, please revert it.

Reply:
removed

Reply:
Thanks!, I just approved the CI to run. If this is green, I think we can merge.

Reply:
@ggrieco-tob a CI job is failing but looks like a flake that is unrelated to my changes, can you restart the job?

Reply:
Oh, that's ok. It is your fault, but some link that started to fail.